rule/rule_test.go

```diff
@@ -358,6 +365,13 @@ func TestShouldKeepExpr(t *testing.T) {
 			src: `
 # keep
 "s"
+`,
+			want: true,
+		}, {
+			desc: "before with description",
+			src: `
+# keep: we need it for the ninja feature
```

Comment:
Could you also add negative tests for the undesirable cases I mentioned in my comments?

Reply:
Added in dc7aa9c.